lib/cdo/rack/request.rb

```diff
@@ -101,6 +102,15 @@ def user_id_from_session_cookie
     rescue
       return nil
     end
+
+    def country
+      env['HTTP_CLOUDFRONT_VIEWER_COUNTRY'] ||
+        location&.country_code
+    end
+
+    def gdpr?
+      gdpr_country_code?(country)
```

Comment:
Apparently we've been using EEA as a broader catch-all, rather than EU.  Maybe we could have the option to query either?  @poorvasingal can comment fully.

Reply:
I've just been trying to use EEA to be more broad / play it safer. EEA = EU + Iceland, Liechtenstein and Norway.

Reply:
updated